agent/ecscni/namespace_helper.go

```diff
@@ -16,9 +16,10 @@ package ecscni
 import (
 	"context"
 
+	current "github.com/containernetworking/cni/pkg/types/100"
```

Comment:
https://github.com/aws/amazon-ecs-cni-plugins/commit/f216cc7bd251dcef00b0c94951e3702c3fbe6ae0

We'll need to validate that the updated version doesn't break any of our calls.  The above change in the cni-plugins library was reverted because the api contract changed between versions, which meant that the delete function for IPAM cleanup was not being executed.

Reply:
+1. It would be great if you add the Testing Done section to the PR. The DEL command of the CNI plugin can fail silently. It would be nice to make sure there are no failures for an awsvpc task with/without trunking.

Reply:
Testing has been completed with the latest changes to the PR (both commits), and we have verified that there were no errors with the DEL command (nor any other failures.) ✅